modular_skyrat/modules/aesthetics/clothing/clothing.dm

```diff
@@ -1,3 +1,72 @@
+
+///////////////////////////////////////////////////////////////////////SHOES
+
+/obj/item/clothing/shoes/sneakers/white
+	worn_icon = 'modular_skyrat/master_files/icons/mob/clothing/feet.dmi'
```

Comment:
Remove that, shoes are using GAGS so that will just do nothing.

Reply:
fiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiiine